frontend/spa-assemble-config.json

```diff
@@ -41,6 +41,7 @@
     "@openmrs/esm-ward-app": "next",
     "@openmrs/esm-stock-management-app": "next",
     "@openmrs/esm-billing-app": "next"
```

Comment:
```suggestion
    "@openmrs/esm-billing-app": "next",
```

Reply:
@denniskigen right, my bad. It has been already corrected